Source/DafnyCore/Verifier/Translator.ExpressionTranslator.cs

```diff
@@ -1919,6 +1919,7 @@ public ExpressionTranslator(ExpressionTranslator etran, Boogie.Expr heap)
               || attr.Name == "timeLimitMultiplier"  // This is a Dafny-specific attribute
               || (attr.Name == "timeLimit" && hasNewTimeLimit)
               || (attr.Name == "rlimit" && hasNewRLimit)
+              || (attr.Name == "revealed_fn")
```

Comment:
Could you change this attribute name to something more explicit, such as "revealedFunction" ?

Reply:
For sure.